client/login/magic-login/style.scss

```diff
@@ -153,23 +156,10 @@
 
 .magic-login__footer {
 	margin: 4px 0 48px 0;
+	text-align: center;
 
 	a {
-		border-bottom: 1px solid #c8d7e1;
-		color: var(--studio-gray-90);
-		display: block;
 		font-size: $font-body-small;
-		font-weight: 600;
-		padding: 0 24px;
-		text-align: center;
-		text-decoration: underline;
-
-		&:hover {
-			color: var(--color-primary);
-		}
-		&:last-of-type {
-			border-bottom: none;
-		}
 
```

Comment:
I can't find an instance where there are 2 anchors in a magic login footer, so these border styles seem safe to remove.